reply for connection

simsinght opened the pull request:
### Merge Readiness: Not Yet
Things to do before merging
- [] Go through code to make sure it follows existing gather/expo conventions
- [] Test on a device (simulator seems to have slightly different behavior)
- - mainly the the block doesn't bounce back into place while on iOS it does. in the swipe video you can see the block remains off to the right of the screen
- [] Additional Testing?

### Swipe for Connections
Video of function: 

https://github.com/user-attachments/assets/a5b59d85-fb61-43b4-8172-c57e2aaa64cd

### Reply in Block Menu
Video of function:

https://github.com/user-attachments/assets/a1653c5a-d65f-4830-8881-0ba410fe690f

### Disclaimers
- Vibe coded using cursor free trial. Hence needing to double check the code convention not being atrocious.

simsinght commented on components/BlockTexts.tsx:
```diff
@@ -59,7 +60,7 @@ export const InspoBlocks = [
   // {
   //   content:
-  //     "Collagist’s Note: New York is a swarm of signs and unholy advertisements. I encounter haphazard phrases daily like dropped pennies; I pick them up to store in my pocketbook (iPhone Notes App), safekeeping these found letters that have gone on a walk (Walking is reading. Writing is walking): cruel embankments, necrologists of the newspapers, pompous rivers, sozzled, jealous spaghetti",
+  //     "Collagist's Note: New York is a swarm of signs and unholy advertisements. I encounter haphazard phrases daily like dropped pennies; I pick them up to store in my pocketbook (iPhone Notes App), safekeeping these found letters that have gone on a walk (Walking is reading. Writing is walking): cruel embankments, necrologists of the newspapers, pompous rivers, sozzled, jealous spaghetti",
```
not sure how this snuck in. I'll drop this change.

simsinght commented on components/BlockTexts.tsx:
```diff
@@ -79,48 +80,79 @@ const BlockViewImpl = ({
-    renderRightActions={() => <RightActions />}
-    onSwipeableOpen={(direction, swipeable) => {
-      if (direction === "left") {
-        return;
```
switch out the functionality to be more aligned with this approach of just pushing to the router and implementing a set of "LeftActions" and bringing back the use of RightActions — current Swipeable has gotten too verbose especially with the YStack explicitly put there.

simsinght commented on components/TextForageView.tsx:
```diff
@@ -631,26 +733,7 @@ export function TextForageView({ collectionId }: { collectionId?: string }) {
-                    onSaveResult();
-                  }
-                }}
+                onPress={onSend}
```
hmm way too big of a difference. switch back to using the onSaveResult and maybe instead pass on the collectionIds as an argument? it's only used in this one context so changing it should be fine.

spencerc99 commented:
omg this is great! ill pull this and test soon. i think that looks great just should button up UI

simsinght commented:
this has been soooo slow going. went back to work after creating this just to get fired.

anyways. i'm happier with the state the PR is in now. or at least i've addressed the points i'd left for myself.

not sure what you meant by button up UI? do you think the collection names need to be bigger above the input? larger x button?
